### PR TITLE
fix(content): Tweak remnant ground assualt bounty job

### PR DESCRIPTION
**Pull request by McloughlinGuy**
## Summary
So I was doing the "Korath ground assault bounty" and I destroyed the ship like usual and I noticed that even though I destroyed the ship the mission wasn't completing. I then realized I hadn't destroyed the fighters. The on visit message was just the generic one so I didn't realize what I'd done wrong until I got there. I decided to update the description and add a custom "on visit" message to clarify a little more that you need to destroy the fighters.

## Alternative Approach
Make killing the fighters not required to complete the mission or use the "generic fleet bounty on visit" message instead of the current "generic bounty on visit" since its technically a fleet.

### CODE REVIEW

**Review by Zitchas — APPROVED** [review]
Looks like a good fix. Thanks for catching that!

**Review by Saugia — CHANGES_REQUESTED** [review]
Use "it's" for "and any fighters its carrying", because "its" is referring to possession, and "it's" is condensed "it is". So, "any fighters *it is* carrying.

**Comment by warp-core** [comment]
> Use "it's" for "and any fighters its carrying", because "its" is referring to possession, and "it's" is condensed "it is". So, "any fighters _it is_ carrying".

I think Zitchas prefers to have no contractions in Remnant stuff, so it should be `it is`.

File: data/remnant/remnant jobs.txt [hunk 1]
```diff
@@ -200,7 +200,7 @@ mission "Remnant: Bounty 5"
 	job
 	repeat
 	name "Destroy Ground Assault Bounty"
-	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it, and then return to <planet> to receive your payment of <payment>."
+	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it and any fighters its carrying, and then return to <planet> to receive your payment of <payment>."
```
Comment (warp-core) [inline data/remnant/remnant jobs.txt:203]:
```suggestion
	description "A Korath ship is hunting for our outposts in the Ember Wastes. Track it down, eliminate it and any fighters it is carrying, and then return to <planet> to receive your payment of <payment>."
```

Comment (McloughlinGuy) [reply data/remnant/remnant jobs.txt]: I accidentally did this exact thing before realizing that you suggested that lmao

**Review by Saugia — APPROVED** [review]

**Pull request merged**